src/main/java/fi/evident/raudikko/Analysis.java

```diff
@@ -277,6 +289,7 @@ public String toString() {
                 ", malagaVapaaJalkiosa=" + malagaVapaaJalkiosa +
                 ", possibleGeographicalName=" + possibleGeographicalName +
                 ", requireFollowingVerb='" + requireFollowingVerb + '\'' +
+                ", baseFormParts='" + (baseFormParts != null ? String.join(", ", baseFormParts) : null) + '\'' +
```

Comment:
Listan toString on itsessään mainio, joten tää voi olla vain:

```suggestion
                ", baseFormParts='" + baseFormParts + '\'' +
```